test/middleware.spec.js

```diff
@@ -30,7 +30,14 @@ describe('Middleware', function() {
     });
 
     beforeEach(function(done) {
-      client.del('ratelimit:127.0.0.1', done);
+      // cleanup Redis keys
+      // TODO: cleanup is coupled to how the default "ip" shorthand works
+      var server = express();
```

Comment:
took me a while to figure out why you were doing this
would it be more intuitive to just delete all keys that start with `ratelimit:`?